routing.rst

```diff
@@ -475,6 +475,13 @@ that are special: each adds a unique piece of functionality inside your applicat
 ``_format``
     Used to set the request format (:ref:`read more <routing-format-param>`).
 
+``_fragment``
+
+ .. versionadded:: 3.2
```

Comment:
I know it's a bit verbose, but in Symfony Docs the `versionadded` directive is used like this:

```
.. versionadded:: 3.2
    The ``_fragment`` parameter was introduced in Symfony 3.2.
```